cucumber/authenticators/features/support/authn_oidc_helper.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+#
+# Utility methods for OIDC authenticator
+#
+module AuthnOidcHelper
+  include AuthenticatorHelpers
+
+  # relevant for original oidc flow
+  # def login_with_oidc(service_id:, account:)
+  #   path = "#{conjur_hostname}/authn-oidc/#{service_id}/#{account}/login"
+  #   payload = { code: oidc_auth_code, redirect_uri: oidc_redirect_uri }
+  #   post(path, payload)
+  #   @login_oidc_conjur_token = @response_body
+  # end
+
+  # # relevant for oidc flow for Conjur oidc token retrieved in oidc login flow
+  # def authenticate_conjur_oidc_token_with_oidc(service_id:, account:)
+  #   path = "#{conjur_hostname}/authn-oidc/#{service_id}/#{account}/authenticate"
+  #   # TODO: Since the input going to change to a base64 signed token, i didnt invest time to extract the real values
+  #   payload = { id_token_encrypted: "login_oidc_conjur_token", user_name: "alice", expiration_time: "1231" }
+  #   post(path, payload)
+  # end
+
+  def authenticate_id_token_with_oidc(service_id:, account:)
+    path = "#{conjur_hostname}/authn-oidc/#{service_id}/#{account}/authenticate"
+    # TODO: Enable the following comment once the production code knows to get encoded id_token
+    # payload = { id_token: "#{oidc_id_token}" }
+    payload = { id_token: "{\"preferred_username\": \"alice\",\"email\": \"alice@conjur.net\"}" }
+    post(path, payload)
+  end
+
+  def oidc_authorization_code
+    path_script = "/authn-oidc/phantomjs/scripts/fetchAuthCode"
+    authorization_code_file = "cat /authn-oidc/phantomjs/scripts/authorization_code"
+
+    system("sh #{path_script}")
+    @oidc_auth_code = `#{authorization_code_file}`
+  end
+
+  def fetch_oidc_id_token
+    path = "#{oidc_provider_internal_uri}/token"
+    payload = { grant_type: 'authorization_code', redirect_uri: oidc_redirect_uri, code: oidc_auth_code }
+    options = { user: oidc_client_id, password: oidc_client_secret }
+    execute(:post, path, payload, options)
+    oidc_id_token
+  end
+
+  def set_oidc_variables
+    path = "cucumber:variable:conjur/authn-oidc/keycloak"
+    Secret.create resource_id: "#{path}/provider-uri", value: oidc_provider_uri
+    Secret.create resource_id: "#{path}/id-token-user-property", value: oidc_id_token_user_property
```

Comment:
Use parens in the two statements above to conform to ruby style guide

Reply:
Fix in pr #925